notebooks/colab_metl_predicting.ipynb

```diff
@@ -0,0 +1,2036 @@
+        "cellView": "form"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "Downloading: \"https://zenodo.org/records/11051645/files/METL-L-2M-3D-GB1-epegcFiH.pt?download=1\" to /root/.cache/torch/hub/checkpoints/epegcFiH.pt\n",
+            "100%|██████████| 9.40M/9.40M [00:01<00:00, 7.44MB/s]\n"
+          ]
+        },
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Initialized PDB bucket matrices in: 0.000\n",
+            "Initialized PDB bucket matrices in: 0.000\n"
+          ]
+        }
+      ],
+      "source": [
+        "# @title Available metl models\n",
+        "# @markdown You may use this dropdown here to chose a metl model to predict with. Running this cell will load the selected model.\n",
+        "metl_model = 'metl-l-2m-3d-gb1' # @param [\"metl-g-20m-1d\",\"metl-g-20m-3d\",\"metl-g-50m-1d\",\"metl-g-50m-3d\",\"metl-l-2m-1d-gfp\",\"metl-l-2m-3d-gfp\",\"metl-l-2m-1d-dlg4\",\"metl-l-2m-3d-dlg4\",\"metl-l-2m-1d-gb1\",\"metl-l-2m-3d-gb1\",\"metl-l-2m-1d-grb2\",\"metl-l-2m-3d-grb2\",\"metl-l-2m-1d-pab1\",\"metl-l-2m-3d-pab1\",\"metl-l-2m-1d-tem-1\",\"metl-l-2m-3d-tem-1\",\"metl-l-2m-1d-ube4b\",\"metl-l-2m-3d-ube4b\",\"metl-bind-2m-3d-gb1-standard\",\"metl-bind-2m-3d-gb1-binding\",\"metl-l-2m-1d-gfp-ft-design\",\"metl-l-2m-3d-gfp-ft-design\"]\n",
+        "metl.load_from_ident(metl_model)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "UENVWR2srLGD"
+      },
+      "source": [
+        "Depending on the model chosen, different files might be needed. This example is setup to use metl-l-2m-3d-gb1 and will need a few data for prediction. **Link to the model descriptions here again to describe the IO of each model**.\n",
+        "\n",
+        "Specifically, for this 3d gb1 model we will need:\n",
+        "- A wild type sequnece\n",
+        "- a PDB structure file (as this is a 3d model)\n",
+        "- variants to use with METL"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "HBAocw_urLGE",
+        "cellView": "form"
+      },
+      "outputs": [],
+      "source": [
+        "# @title Protein wild type\n",
+        "# @markdown Enter the wild type of your protein here. The wildtype for gb1 is provided to use with the default model example here.\n",
+        "wildtype = 'MQYKLILNGKTLKGETTTEAVDAATAEKVFKQYANDNGVDGEWTYDDATKTFTVTE' # @param {type:\"string\", placeholder:\"Enter a wildtype here\"}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9H12FqxWrLGF"
+      },
+      "source": [
+        "The PDB file is something that we will reference from within the colab through its file path. If you don't need a PDB file for your use-case, simply set this variable to None."
```

Comment:
```suggestion
        "The PDB file is something that we will reference from within the colab through its file path. If you don't need a PDB file for your metl model, simply set this variable to None."
```

Reply:
I deleted this. I made it so the user doesn't have to think of that but forgot to get rid of the previous instructions before the dropdown was available